swagger.json

```diff
@@ -5850,6 +6001,72 @@
             "items": {
                 "$ref": "#/definitions/DocumentType"
             }
+        },
+        "User": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "The id of the User record.",
+                    "example": 123
+                },
+                "name" : {
+                    "type": "string",
+                    "description": "The first and last name of the user.",
+                    "example": "Jane Doe"
+                },
+                "email": {
+                    "type": "string",
+                    "description": "The email address of this user.",
+                    "example": "user@company.com"
+                },
+                "organizationRole" : {
+                    "type": "string",
+                    "description": "The name of the role the user is assigned to for the organization. This will be blank for users that only have access to specific tags.",
+                    "example": "Read-Only Admin"
+                },
+                "tagRoles" : {
+                    "type": "array",
+                    "description": "The specific tags this user has access to. This will be blank for users that have full access to the organization.",
+                    "items": {
+                        "$ref": "#/definitions/UserTagRole"
+                    }
+                },
+                "authType": {
+                    "type": "string",
+                    "enum": [
+                        "basic",
+                        "saml"
+                    ],
+                    "description": "The authentication type the user uses to authenticate. To use SAML your organization must have a configured SAML integration."
+                }
+            }
+        },
+        "UserTagRole": {
+            "type": "object",
+            "properties": {
+                "tag": {
+                    "type": "object",
+                    "properties": {
+                        "id": {
+                            "type": "integer",
+                            "format": "int64",
+                            "description": "The ID of this tag.",
+                            "example": 12345
+                        },
+                        "name": {
+                            "type": "string",
+                            "description": "Name of this tag.",
+                            "example": "North Carolina Distribution"
+                        }
+                    }
+                },
+                "role": {
+                    "type": "string",
```

Comment:
Could this be an enum?

Reply:
Just thinking out loud -- would it make sense to specify all of the standard role types as an enum, with an additional `CustomRole` type as an option, that acts as a discriminator for another `customRole` field?

Reply:
Do you think this would lead to cleaner error handling?

Reply:
I'm primarily thinking about our generated SDKs (which I plan to start generating out soon). If these magic values are encoded in the swagger schema, then the SDKs will have a set of auto-complete options.